addons/l10n_lt/data/account_fiscal_position_template_data.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo noupdate="1">
+    <!-- Account Fiscal Position Template -->
+    <record id="account_fiscal_position_template_lt" model="account.fiscal.position.template">
+        <field name="name">LT</field>
+        <field name="auto_apply" eval="True"/>
+        <field name="vat_required" eval="True"/>
+        <field name="country_id" ref="base.lt"/>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="sequence" eval="10"/>
+    </record>
+    <record id="account_fiscal_position_template_eu" model="account.fiscal.position.template">
+        <field name="name">EU</field>
+        <field name="auto_apply" eval="True"/>
+        <field name="vat_required" eval="True"/>
+        <field name="country_group_id" ref="base.europe"/>
+        <field name="chart_template_id" ref="account_chart_template_lithuania"/>
+        <field name="sequence" eval="20"/>
+    </record>
+    <record id="account_fiscal_position_template_out_eu" model="account.fiscal.position.template">
+        <field name="name">Not EU</field>
+        <field name="auto_apply" eval="False"/>
```

Comment:
Can be auto_apply True: if it is not from the EU, is not it this fiscal position anyways?

Reply:
I guess it could have auto_apply set to True. Though I didn't understand about second part of your comment. Can you rephrase that ("if it is not from the EU, is not it this fiscal position anyways") ?

Reply:
If it is outside EU, it should always be this fiscal position.